packages/databricks-vscode/src/configuration/types.ts

```diff
@@ -9,6 +9,10 @@ export type DatabricksConfigs = {
     workspaceFsPath?: string;
 };
 
+export type DatabricksConfigSource = {
```

Comment:
Nitpick, but it's a bit confusing how we have `DatabricksConfigs` as a plural thing, even though it's just one object with key value pairs. And in some cases we refer to it in a singular form, but in other cases we refer to its keys as configs. And this source name uses a singular notation now. 

I would expect DatabricksConfigs to be DatabricksConfig (without s), and then many other types and methods that operate on its keys renamed to have a key in their names, not config

Reply:
Makes sense mostly. Not sure I understand the second part. Does it mean that we add the "key" suffix?

Reply:
Yes, for example this renames:
BUNDLE_CONFIGS -> BUNDLE_KEYS
OVERRIDEABLE_CONFIGS -> OVERRIDEABLE_KEYS
OverrideableConfigs -> OverrideableConfig
isOverrideableConfig -> isOverrideableKey
isBundleConfig -> isBundleKey

Reply:
Makes sense. I will take a stab at it. I will push it into the same PR if the changes are not too big.